### PR TITLE
feat: 🔥 Better header and changed search bar location on mobile

### PR DESCRIPTION
**Pull request by tobycm**
2 commit so fire siuuuu

### CODE REVIEW

File: src/components/Header/index.tsx
Comment (pdt1806) [inline src/components/Header/index.tsx:1]: the convo is still there ;-;

Comment (tobycm) [reply src/components/Header/index.tsx:1]: why not

**Comment by tobycm** [comment]
hmm ok i approve ur commit

**Review by pdt1806 — APPROVED** [review]
yes

**Pull request merged**